concourse/pipelines/deploy-cloudfoundry.yml

```diff
@@ -168,13 +232,99 @@ jobs:
               - cf-secrets.yml
               - generate/cf-secrets.yml
 
+  - name: generate-cf-certs
+    plan:
+      - aggregate:
+        - get: paas-cf
+          passed: ['init']
+          trigger: true
+        - get: bosh-CA
+      - task: generate-cf-CAs
+        config:
+          image: docker:///keymon/certstrap#withcurl
```

Comment:
Yes, this must be merged first and the dockerhub image setup: https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/16